src/app/issuer/components/badgeclass-detail/badgeclass-detail.component.ts

```diff
@@ -29,13 +29,19 @@ import { TranslateService } from '@ngx-translate/core';
 import { PageConfig } from '../../../common/components/badge-detail/badge-detail.component.types';
 import { PdfService } from '../../../common/services/pdf.service';
 import { QrCodeApiService } from '../../services/qrcode-api.service';
-
+import { toDataURL } from 'qrcode';
```

Comment:
```qrcode``` is a dependency from ```angularx-qrcode``` and not explicitly listed in our ```package.json```. If we need to directly import from ```qrcode``` i think it would be better to install it and have it in our ```package.json```as well.

Reply:
added it to package.json